spec/models/calculator/weight_spec.rb

```diff
@@ -2,13 +2,13 @@
 
 describe Calculator::Weight do
   it "computes shipping cost for an order by total weight" do
-    variant1 = double(:variant, weight: 10)
-    variant2 = double(:variant, weight: 20)
-    variant3 = double(:variant, weight: nil)
+    variant1 = build(:variant, weight: 10)
+    variant2 = build(:variant, weight: 20)
+    variant3 = build(:variant, weight: nil)
 
-    line_item1 = double(:line_item, variant: variant1, quantity: 1)
-    line_item2 = double(:line_item, variant: variant2, quantity: 3)
-    line_item3 = double(:line_item, variant: variant3, quantity: 5)
+    line_item1 = build(:line_item, variant: variant1, quantity: 1)
+    line_item2 = build(:line_item, variant: variant2, quantity: 3)
+    line_item3 = build(:line_item, variant: variant3, quantity: 5)
```

Comment:
These changes from double to build are needed because "object.is_a?(Spree::LineItem)" will fail for a double.